client-common/src/main/java/com/cloudera/livy/client/common/ClientConf.java

```diff
@@ -214,11 +211,12 @@ private void check(boolean test, String message, Object... args) {
 
   /** Logs a warning message if the given config key is deprecated. */
   private void logDeprecationWarning(String key) {
-    DeprecatedConf altConfs = getConfigsWithAlternatives().get(key);
-    if (altConfs != null) {
+    String altConfKey = allAlternativeKeys().get(key);
+    if (altConfKey != null) {
+      DeprecatedConf altConfs = getConfigsWithAlternatives().get(altConfKey);
```

Comment:
Actually on second look this line is unnecessary and `altConfs.key()` on the line below can be replaced with `key` now. This function was previously testing the inverse of what it was supposed to.